src/core/lib/security/credentials/tls/grpc_tls_certificate_provider.h

```diff
@@ -34,6 +35,12 @@
 #include "src/core/lib/security/credentials/tls/grpc_tls_certificate_distributor.h"
 #include "src/core/lib/security/security_connector/ssl_utils.h"
 
+// Known grpc_tls_certificate_provider implementation types
+#define GRPC_TLS_CERTIFICATE_PROVIDER_TYPE_STATIC_DATA "StaticData"
```

Comment:
Similar comment here: I don't think we want these as macros.  We should inline them in the individual provider impls.